src/2.6.0/wso2apim.js

```diff
@@ -247,6 +250,32 @@ function constructAPIDef(user, gatewayEnv, apiDef, apiId) {
   }
 }
 
+function constructCorsConfiguration(apiDef) {
+  const { origins, credentials, headers, methods } = apiDef.cors;
+  const defaultAllowHeaders /* default WSO2 cors config */ = [
+    'authorization',
+    'Access-Control-Allow-Origin',
+    'Content-Type',
+    'SOAPAction',
+    'apikey',
+  ];
+  const defaultAllowMethods /* default WSO2 cors config */ = [
+    'GET',
+    'PUT',
+    'POST',
+    'DELETE',
+    'PATCH',
+    'OPTIONS',
+  ];
+  return {
+    corsConfigurationEnabled: true,
+    accessControlAllowOrigins: origins || ['*'],
+    accessControlAllowCredentials: credentials || false,
```

Comment:
I think it's safe to validate the value assigned to `credentials` (is it boolean?) as a part of `validateConfig()` [here](https://github.com/ramgrandhi/serverless-wso2-apim/blob/990f8895fc3e07c32bce8ea7a05cca528e6ced2d/src/index.js#L153) before assigning it here.